package-manylinux.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+## targeting old glibc
+# docker run --mount type=bind,source="$(pwd)",target="/usr/src/app" -it quay.io/pypa/manylinux2014_x86_64
+
+cd /usr/src/app
+
+for py in $(ls /opt/python/ | grep cp3); do
```

Comment:
This is pretty awkward -- I'd like to build one wheel for many different python versions. Alas